packages/api/srcbook/index.mts

```diff
@@ -217,3 +228,33 @@ export function removeSrcbook(srcbookDir: string) {
 export function removeCodeCellFromDisk(srcbookDir: string, filename: string) {
   return fs.rm(pathToCodeFile(srcbookDir, filename));
 }
+
+export async function updateSessionEnvTypeDeclarations(sessionId: string) {
+  const sessionSecrets = await getSecretsAssociatedWithSession(sessionId);
+  if (!Object.entries(sessionSecrets).length) return;
+  const envTypeDeclarationFileContent = generateEnvTypesFile(sessionSecrets);
+  const srcbookDir = pathToSrcbook(sessionId);
+  const envDtsPath = Path.join(srcbookDir, 'src', 'env.d.ts');
```

Comment:
Nit but i would put this in the repo root. I think that's where I've typically seen these global envs.